draft-ietf-quic-transport.md

```diff
@@ -1131,7 +1219,34 @@ version negotiation occurred but it would have selected a different version
 based on the value of the supported_versions list.
 
 
-## Proof of Source Address Ownership {#source-address-token}
+## Stateless Retries {#stateless-retry}
+
+A server can process an initial cryptographic handshake messages from a client
+without committing any state.  This allows a server to perform address
+validation ({{address-validation}}, or to defer connection establishment costs.
+
+In order to ensure consistent state regarding stream offsets and packet numbers,
+a server that generates a response to an initial packet without retaining
+connection state MUST use the Server Stateless Retry packet
+({{packet-server-stateless}}).  This packet causes a client to reset the state
+of stream 0 and continue the connection attempt with fresh stream state while
+maintaining the state of the cryptographic handshake.
+
+A server that sends a Server Stateless Retry packet is then not obligated to
+ensure that it correctly accounts for data already sent on stream 0 either by
+both itself or by the client.  Similarly, a server also does not have to
```

Comment:
I think it's adequate to say that no connection state needs to be maintained after sending a Stateless Retry... I don't think it's necessary to discuss accounting of data on stream 0 here.